### PR TITLE
Update ndarray to version 0.13

### PR DESCRIPTION
**Pull request by stefan-k**

### CODE REVIEW

File: src/krylov/householder.rs [hunk 1]
```diff
@@ -18,7 +18,8 @@ where
     let alpha = -x[0].mul_real(norm / x[0].abs());
     x[0] -= alpha;
     let inv_rev_norm = A::Real::one() / x.norm_l2();
-    azip!(mut a(x) in { *a = a.mul_real(inv_rev_norm)});
+    // azip!(mut a(x) in { *a = a.mul_real(inv_rev_norm)});
```
Comment (jturner314) [inline src/krylov/householder.rs]:
```suggestion
```

File: src/krylov/mgs.rs [hunk 2]
```diff
@@ -51,7 +51,7 @@ impl<A: Scalar + Lapack> Orthogonalizer for MGS<A> {
         for i in 0..self.len() {
             let q = &self.q[i];
             let c = q.inner(&a);
-            azip!(mut a (&mut *a), q (q) in { *a = *a - c * q } );
+            azip!((a in &mut *a, &q in q) *a = *a - c * q);
```
Comment (jturner314) [inline src/krylov/mgs.rs]:
```suggestion
            azip!((a in a, &q in q) *a = *a - c * q);
```

File: src/krylov/mgs.rs [hunk 3]
```diff
@@ -88,7 +88,7 @@ impl<A: Scalar + Lapack> Orthogonalizer for MGS<A> {
             // Linearly dependent
             return AppendResult::Dependent(coef);
         }
-        azip!(mut a(&mut *a) in { *a = *a / A::from_real(nrm) });
+        azip!((a in &mut *a) *a = *a / A::from_real(nrm));
```
Comment (jturner314) [inline src/krylov/mgs.rs]:
```suggestion
        azip!((a in a) *a = *a / A::from_real(nrm));
```

**Comment by stefan-k** [comment]
Thank you for the suggestions!

**Comment by stefan-k** [comment]
I probably should have tested the suggestions locally instead of simply accepting them on Github ;)

**Comment by jturner314** [comment]
My bad. I'm sorry for the incorrect suggestions. I wasn't entirely sure that removing the `&mut *` would work, but I knew CI would catch any issues. :)

**Comment by jturner314** [comment]
Thanks for this @stefan-k!

**Pull request merged**